src/components/link.js

```diff
@@ -0,0 +1,39 @@
+var registerComponent = require('../core/component').registerComponent;
+
+module.exports.Component = registerComponent('link', {
+  schema: {
+    url: { default: '' },
```

Comment:
Wrapping in `<a>` isn't great. It doesn't fit with the framework, is syntactically more complicated (both in HTML, and you can't just synchronously do `setAttribute('link', ..>)` or use mixins/`aframe-mss`, and will cause tons of complications in introducing a non-A-Frame element into the scene graph.

No strong preference for href/link, but `href` might be confusing because people will be passing a URL string to it, when it needs both an `href` and an event name (`href="href: blah.com; on: grab`).